06-building-a-program-start-to-finish/writing-tests/output/spec.js

```diff
@@ -0,0 +1,83 @@
+
+const invertedIndex = new InvertedIndex();
+const mockData = [
+  {
+    title: 'Alice in Wonderland',
+    text: 'Alice falls rings.'
+  },
+
+  {
+    title: 'The Lord of the Rings: The Fellowship of the Ring.',
+    text: 'An unusual alliance of in the rings.'
+  }
+];
+
+
+describe('Inverted Index Test', () => {
+  describe('It Validate Book Data', () => {
+    it('should return invalid file type if book is not an array of objects',
+     () => {
+       expect(invertedIndex.validateInput('rowland')).toBe(false);
+       expect(invertedIndex.validateInput(2)).toBe(false);
+       expect(invertedIndex.validateInput([1, 2, 3])).toBe(false);
+       expect(invertedIndex.validateInput([{ sex: 'male', age: 18 }]))
```

Comment:
@andela-aolaniran i wrote the test hoping that invalid inputs would fail, but i agree with you.